src/nni_manager/core/test/ipcInterfaceTerminate.test.ts

```diff
@@ -106,7 +106,7 @@ describe('core/ipcInterface.terminate', (): void => {
                 assert.ok(!procError);
                 deferred.resolve();
             },
-            2000);
+            5000);
```

Comment:
why change this?

Reply:
Same reason as above to accommodate the queue change.